_ingest-pipelines/processors/json.md

```diff
@@ -0,0 +1,200 @@
+---
+layout: default
+title: JSON
+parent: Ingest processors
+nav_order: 170
+---
+
+# JSON processor
+
+The `json` processor serializes a string-valued field into a map of maps, which can be useful for various data processing and enrichment tasks.
+
+The following is the syntax for the `json` processor:
+
+```json
+{
+  "processor": {
+    "json": {
+      "field": "<field_name>",
+      "target_field": "<target_field_name>",
+      "add_to_root": <boolean>
+    }
+  }
+}
+```
+{% include copy-curl.html %}
+
+## Configuration parameters
+
+The following table lists the required and optional parameters for the `json` processor.
+
+Parameter | Required/Optional | Description |
+|-----------|-----------|-----------|
+`field` | Required | The name of the field containing the JSON-formatted string to be deserialized.
+`target_field` | Optional | The name of the field where the deserialized JSON data will be stored. If this parameter is not provided, the deserialized data is added to the root of the document.
```

Comment:
For the `target_field`, if it's not specified, the value will be stored in the `field` field, and if the specified target_field exists, it will be overwritten.